wemake_python_styleguide/violations/consistency.py

```diff
@@ -2415,3 +2415,49 @@ class NotInWithUnaryOpViolation(ASTViolation):
 
     error_template = 'Found legacy `not ... in`, use `... not in` instead'
     code = 364
+
+
+@final
+class SimplifiableMatchViolation(ASTViolation):
+    """
+    Some ``match`` statements can be simplified to ``if`` statements.
+
+    Reasoning:
+        Using ``match`` for simple two-case conditions
+        (including wildcard ``_``)
+        is unnecessarily verbose and less performant than a plain ``if/else``.
+        Simple conditions should prefer readability and simplicity.
+
+    Solution:
+        Replace violating ``match ... case _`` statements with ``if ... else``.
+
+    When is this violation is issued?
```

Comment:
```suggestion
    When is this violation is raised?
```

we use "raise" in this context